src/test_driver/esp32/qemu_setup.sh

```diff
@@ -34,7 +34,6 @@ die() {
 cd "$here" || die 'ack!, where am I?!?'
 
 # shellcheck source=/dev/null
-source idf.sh
 rm -f ./build/sdkconfig
-SDKCONFIG=./build/sdkconfig SDKCONFIG_DEFAULTS=sdkconfig_qemu.defaults idf make defconfig
-SDKCONFIG=./build/sdkconfig idf make -j8 esp32_elf_builder
+SDKCONFIG=./build/sdkconfig SDKCONFIG_DEFAULTS=sdkconfig_qemu.defaults idf.py build defconfig
```

Comment:
This won't work. As this command is not valid for CMake `idf.py build defconfig`

Reply:
Yes, I checked it locally. It is removed.